packages/11ty/_plugins/figures/iiif/manifest/index.js

```diff
@@ -145,6 +145,15 @@ module.exports = class Manifest {
     })
     try {
       this.json = builder.toPresentation3(manifest)
+      /**
+       * Note: Builder can't currently create annotations with target regions,
+       * so set the target on each annotation directly
+       */
```

Comment:
Should we add a link to the issue? https://github.com/stephenwf/iiif-builder/issues/6
```suggestion
      /**
       * Note: Builder can't currently create annotations with target regions,
       * so set the target on each annotation directly
       * @see @stephenwf/iiif-builder#6
       */
```